open-machine-learning-jupyter-book/deep-learning/gan.ipynb

```diff
@@ -3,6 +3,7 @@
   {
```

Comment:
<p>The output is so long, let's see if we can shorten it with a roller?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>